### DISCUSSION TITLE
Delete an "organization"

### DISCUSSION THREAD

**Discussion by ccasadei** [langchain-ai/langsmith-sdk#428] [Ideas]
### Feature request

In the "**Manage Organization**" section I would need a way to delete organizations that are no longer needed.

### Motivation

Clean up the list of organizations that I created as a trial or that I no longer manage.

**Comment by hinthornw** [comment]
On the roadmap :)

**Reply by lomohuang** [reply]
Is there any update currently?

**Comment by ccasadei** [comment]
Maybe this can be done in two stages:

- first stage: disable an organization (keeping all data). A disabled organization can be re-enabled in needed. The state (enabled/disabled) can be also a filter for visualization.

- second stage: delete a DISABLED organization (deleting all data)

**Comment by hinthornw** [comment]
Currently org deletion is done by emailing support.langchain.com
